.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+  module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint')],
+  rules: {
+    'notice/notice': 'off',
+    'react-hooks/exhaustive-deps': 'off',
```

Comment:
I think we should keep these two on. If we really need to turn them off for a particular usage we can make that decision on a case by case basis.

Reply:
You are right, I rewritten the code to works with those rules. Actually only one rule was broken (react-hooks/exhaustive-deps)
other was just a cope paste from a previous plugin. My bad. 
To be honest I am not a fan of this eslint exhaustive-deps rule (even if it is recommended) because sometimes it causes weird loops :)

Reply:
> Looks great overall. A couple of things just from the Loom:
> 
> * Could we use a different Icon for rebuild and refresh? And maybe a tooltip for each?
> * On the single build page there's a double line of color, presumably one for the overall status and one for each step? What about just using the color for each step and an maybe just an icon in the header section? And also can we remove the padding /margin around the steps?
> * Is it possible to get a link to the source too as well as buildkite? And if possible the head commit message as well as the ID? (though I'm not sure about the second one as that's likely to be the same as the build name in buildkite?)
> 
> @dtuite agree?

@iain-b made a quick Loom preview how it might looks with simpler UI as you suggested and I think it's way better 👍 
https://www.loom.com/share/87559b50666b41e987d04e13b6e9fbb8

Head commit message is same as build name.